challenges/qr_code_component/index.html

```diff
@@ -1,62 +1,69 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>
-      QR code component &bullet; Frontend Mentor Challenges &bullet; Optimum BH
-      Internship
+      QR code component • Frontend Mentor Challenges • Optimum BH Internship
     </title>
+
+    <meta charset="UTF-8" />
+    <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
     <link
+      href="images/apple-touch-icon.png"
       rel="apple-touch-icon"
       sizes="180x180"
-      href="../images/apple-touch-icon.png"
     />
     <link
+      href="images/favicon-32x32.png"
       rel="icon"
-      type="image/png"
       sizes="32x32"
-      href="../images/favicon-32x32.png"
+      type="image/png"
     />
     <link
+      href="images/favicon-16x16.png"
       rel="icon"
-      type="image/png"
       sizes="16x16"
-      href="../images/favicon-16x16.png"
+      type="image/png"
     />
-    <link rel="manifest" href="../site.webmanifest" />
-    <link rel="stylesheet" href="style.css" />
+    <link href="site.webmanifest" rel="manifest" />
+    <link href="style.css" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com" rel="preconnect" />
+    <link crossorigin href="https://fonts.gstatic.com" rel="preconnect" />
   </head>
-  <body class="m-0">
-    <div
-      class="flex h-[100vh] w-full flex-col place-content-center text-center"
-    >
-      <div>
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://www.frontendmentor.io/challenges/qr-code-component-iux_sIO_H"
-          target="_blank"
-          >Frontend Mentor challenge</a
-        >
-      </div>
-      <div>
-        for
-        <a
-          class="text-optimum-blue hover:text-optimum-darkblue"
-          href="https://github.com/optimumBA"
-          target="_blank"
-          >Optimum BH</a
-        >
-        Internship.
-      </div>
+  <body class="m-0 flex h-screen flex-col bg-blue-100">
+    <div class="flex flex-grow items-center justify-center">
+      <div
+        class="mx-3 flex flex-col rounded-lg bg-white px-3 py-5 md:mx-0 lg:w-[20%]"
+      >
+        <img
+          alt="logo"
+          class="contain max-w-full rounded-lg"
+          src="../images/image-qr-code.png"
+        />
+
+        <div class="mt-6 w-full">
+          <label
+            ><a
+              href="https://www.frontendmentor.io/challenges/qr-code-component-iux_sIO_H"
+              >Frontend Mentor challenge</a
+            ></label
+          >
+          <label
+            >for
+            <a href="https://github.com/optimumBA">Optimum BH</a> Internship
+            <a
+              href="https://github.com/optimumBA/frontend_mentor_challenges/tree/qr_code_component/challenges/qr_code_component"
```

Comment:
Put your username instead of `optimumBA` here so that visitors can access your code.